Sources/SwiftDocC/Model/Section/Sections/PossibleValuesSection.swift

```diff
@@ -0,0 +1,82 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2021-2024 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+import SymbolKit
+
+public struct PossibleValuesSection {
+    public static var title: String {
+        return "Possible Values"
+    }
+    
+    /// The list of possible values.
+    public let possibleValues: [PossibleValueTag]
```

Comment:
Question: are these the values that have documentation or the values that are possible?

Reply:
The values that are possible